README.md

```diff
@@ -5,23 +5,291 @@
 
 # Istio Operator
 
-Update 4/22/19
-
-This repo is for the initial, alpha version of the official Istio install operator. It's currently only open to the core 
-design team for commits, while we merge prototypes that evolved in parallel. Once we have
-a basic functional skeleton in place we'll open the repo up for community contributions.
-
-The initial goals/MVP for the operator are:
-
-1.   Replace helm tiller as the recommended install upgrade controller. We will continue to use helm templates and
-the helm library to generate manifests under the hood, but will take ownership of the logic behind the install API.
-The operator will have modes of operation analogous to tiller - controller in and out of cluster and one-shot manifest
-generation for use with external toolchains.
-1.   Provide a structured API to eventually replace the de-facto values.yaml API. This API will be partially moved
-to CRDs used to configure Istio components directly, hence the operator API will only contain the portions that will 
-remain in the operator CRD, and continue to use values.yaml for the parameters that will be moved to component CRDs.
-The API will be aligned with Istio a-la-carte/building blocks philosophy and have a functional structure.
-1.   Add an overlay mechanism to allow users to customize the rendered manifest in a way that survives upgrades. 
-1.   Support upgrade, both in-place and dual control plane version rollover. The latter will be supported through the 
-use of the [istio/installer](https://github.com/istio/installer) charts.
-1.   Provide validation of all parameters.
+## Introduction
+
+The Istio operator CLI is now stable enough for developers to evaluate and experiment with. We welcome your
+contributions - pick an [unassigned open issue](https://github.com/istio/istio/issues?q=is%3Aissue+is%3Aopen+label%3Aarea%2Fenvironments%2Foperator+no%3Aassignee),
+create a feature proposal, or come to the weekly Environments Working Group meeting if you have ideas.
+
+-  [Bugs and feature requests](https://github.com/istio/operator/blob/master/BUGS-AND-FEATURE-REQUESTS.md)
+-  [Contributing guidelines](https://github.com/istio/operator/blob/master/CONTRIBUTING.md)
+-  [Working groups](https://github.com/istio/community/blob/master/WORKING-GROUPS.md)
+
+### Background
+
+We reorganized the current [helm installation parameters](https://istio.io/docs/reference/config/installation-options/) into two groups:
+
+-  The new [platform level installation API](https://github.com/istio/operator/blob/master/pkg/apis/istio/v1alpha2/istiocontrolplane_types.proto), for managing
```

Comment:
md format is 1 space after `-`., not two.  Two spaces for whatever reason confuses many parsers  You might consider running this README.md by a linter.  A 1 time lint could easily be done by copying it over a file in the docs repo and running the linter there locally.

Reply:
I installed and ran mdl (which is what istio.io uses) and it now runs clean apart from line length which I'm not going to deal with until I can get a good MD editor.
btw. I changed the number of spaces although that is not flagged by the linter.